homeassistant/components/analytics/const.py

```diff
@@ -0,0 +1,86 @@
+"""Constants for the analytics integration."""
+from datetime import timedelta
+from enum import Enum
+import logging
+
+ANALYTICS_ENPOINT_URL = "https://floral-resonance-f63b.ludeeus.workers.dev"
+DOMAIN = "analytics"
+INTERVAL = timedelta(days=1)
+STORAGE_KEY = "core.analytics"
+STORAGE_VERSION = 1
+
+
+LOGGER: logging.Logger = logging.getLogger(__package__)
+
+ATTR_ADDON_COUNT = "addon_count"
+ATTR_ADDONS = "addons"
+ATTR_AUTOMATION_COUNT = "automation_count"
+ATTR_AUTO_UPDATE = "auto_update"
+ATTR_DIAGNOSTICS = "diagnostics"
+ATTR_HEALTHY = "healthy"
+ATTR_HUUID = "huuid"
+ATTR_INSTALLATION_TYPE = "installation_type"
+ATTR_INTEGRATION_COUNT = "integration_count"
+ATTR_INTEGRATIONS = "integrations"
+ATTR_ONBOARDED = "onboarded"
+ATTR_PREFERENCES = "preferences"
+ATTR_PROTECTED = "protected"
+ATTR_SLUG = "slug"
+ATTR_STATE_COUNT = "state_count"
+ATTR_SUPERVISOR = "supervisor"
+ATTR_SUPPORTED = "supported"
+ATTR_USER_COUNT = "user_count"
+ATTR_VERSION = "version"
+
+
+class AnalyticsPreference(str, Enum):
+    """Analytics prefrences."""
+
+    BASE = "base"
+    DIAGNOSTICS = "diagnostics"
+    STATISTICS = "statistics"
+    USAGE = "usage"
+
+
+INGORED_DOMAINS = [
```

Comment:
Suggestions to add:

- `conversation`
- `stt`

Reply:
What's the criteria for these domains? It would be good to have a way to avoid needing to remember to update the list here in the analytics integration when we add new integrations that fit the criteria.

Reply:
IMO it should contain integrations that "everyone" has, there is no need to report on those.
So all entity domains and helper integrations should be ignored.

Reply:
Well, it depends. Have entity components reported say something about, e.g., % of people that have at least a `humidifier`. Not sure if that is interesting data.

Reply:
sure